src/test/java/uk/nhs/prm/e2etests/tests/RepositoryE2ETests.java

```diff
@@ -260,7 +255,7 @@ void shouldVerifyThatALargeEhrXMLIsUnchanged() {
         inboundQueueFromMhs.sendMessage(continueRequest, outboundConversationId);
 
         // get all message fragments from gp2gp-messenger observability queue and compare with inbound fragments
-        List<SqsMessage> allFragments = gp2gpMessengerQueue.getAllMessageContaining("COPC_IN000001UK01");
+        List<SqsMessage> allFragments = gp2gpMessengerQueue.getAllMessageContaining("ehr/continue-request");
```

Comment:
Probably this one should remain as "COPC_IN000001UK01" ? I think it is checking the message contain the substring "COPC_IN000001UK01" rather than reading a file

Reply:
+1

Reply:
Comment Addressed!